doc/pages/buffers.asciidoc

```diff
@@ -0,0 +1,48 @@
+= Buffers
+
+== Commands
+
+To open buffers or navigate through the buffers list
+see <<commands#files-and-buffers,`:doc commands files-and-buffers`>>.
+
+== Scratch Buffers
+
+Scratch buffers are useful for volatile data and quick prototyping.
+
+One particular scratch buffer, named *\*scratch**, is automatically created
+when there are no other buffers left in the current session. (which is also
+the case at Kakoune's startup when no files to open have been provided)
+
+You can create you owns by using the `-scratch` parameter of the `:edit` command.
+
+== Debug Buffers
+
+Debug buffers are used to gather diagnostics. They are not accessible while cycling
+over the buffers list.
+
+A specific *\*debug** buffer is used by Kakoune to write error or warning.
+This is also where the ouput of the `:debug` and the `:echo -debug` commands will land.
+
+You can create you owns by using the `-debug` parameter of the `:edit` command.
+
+== FIFO Buffers
+
+The `:edit` command can take a `-fifo` parameter:
+
+---------------------------------------------
+:edit -fifo <filename> [-scroll] <buffername>
+---------------------------------------------
+
+In this case, a buffer named `<buffername>` is created which reads its content
+from fifo `<filename>`. When the fifo is written to, the buffer is automatically
+updated.
+
+If the `-scroll` switch is specified, the initial cursor position will be made
+such as the window displaying the buffer will scroll as new data is read.
```

Comment:
"made such as" sounds a bit awkward, how about dropping the bit about the cursor and just saying "If the `-scroll` switch is specified, the window displaying the buffer will scroll so that the newest data is always visible."